hypershift-operator/controllers/hci/aws_infra.go

```diff
@@ -0,0 +1,169 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
can we please name the folder consistently with package and use full name as for other controllers, i.e `controllers/hostedclusterinfrastructure`?

Reply:
Then aws_infra can just be aws.go

Reply:
Sure. I was going with a suggestion above, when I used `hci`

Reply:
fixed